src/display/api.js

```diff
@@ -2473,6 +2471,9 @@ class WorkerTransport {
         warn(`getRenderingIntent - invalid annotationMode: ${annotationMode}`);
     }
 
+    if (isEditing) {
+      renderingIntent += RenderingIntentFlag.IS_EDITING;
```

Comment:
Should we perhaps call it `RenderingIntentFlag.EDITING` instead, given that for the "isOpList" case below we also call it `OPLIST` instead of `IS_OPLIST`? Looking at the other keys in this enum it might also be more consistent with those.

Reply:
I don't think so, since `OPLIST` is one of the primary intents and this one is more of a "secondary" one where being consistent with the existing `isEditing`-value used e.g. in the public API and the viewer as well seems worthwhile.